recipes/elastic-apm/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "elastic-apm" %}
+{% set version = "6.1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/elastic-apm-{{ version }}.tar.gz
+  sha256: 7e5669556d9a228d5508c3f4925e3a7efd0cbf9203a4655e287408aa6bc53a84
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
I guess you're not actually using a compiler?
```suggestion
```

Reply:
Thanks for the pointer. Actually, there is a C file in the package: https://github.com/elastic/apm-agent-python/blob/836905a97e029bba76043fb3310e972e4cb48c5d/setup.py#L133-L142. @beniwohli could you help?

If the file needs to be compiled, this is probably no candidate for a noarch package, right?

Reply:
yes, the C file is from the [wrapt](https://github.com/GrahamDumpleton/wrapt) library that we vendored. The compiled component is optional and does provide some performance benefits (we haven't tested in a while how much the benefit is, but it's probably quite minor for our use case).

Reply:
> that we vendored

Can you just depend on the conda-forge wrapt package: https://github.com/conda-forge/wrapt-feedstock

Reply:
> > that we vendored
> 
> Can you just depend on the conda-forge wrapt package: https://github.com/conda-forge/wrapt-feedstock

We plan to un-vendor wrapt at some point, but it's not on our immediate roadmap due to other higher priority work :(

Reply:
Ok. As I understand, we just need to decide whether we want to distribute the package with the compiled component as platform-dependent packages or as a platform-independent noarch package without compiling the optional C file.

I am fine with both options. By publishing a noarch package we would partly anticipate un-vendoring wrapt.

What is your opinion? I do not know enough about conda-forge's policies and neither about what's the best choice from the users' and maintainers' point of view.

Reply:
Follow up question: What triggers the compilation step?

Reply:
@tobiasraabe yes, compiler and required header files (IIRC, only `Python.h`). It can be disabled completely by setting the environment variable `ELASTIC_APM_WRAPT_EXTENSIONS` to `false`. IMO, that might be a viable option here so we can get a noarch build.

Reply:
@chrisburr can we neglect the failing windows build? I am not sure but aren't noarch packages only build on linux?